run.py

```diff
@@ -89,18 +150,20 @@ def process(message,firstname,username,chatid):
 					("" if monikers_dict.get(message[3], 0) == 0 else f" ({str(amount)} doge)") +
 					" to @"+person+"",chatid)
 			(balance, pending_balance) = returnBal(person)
-			sendMsg("@"+person+" Balance : "+math.floor(balance)+ "Doge ("+math.floor(pending_balance)+" Doge)",chatid)
+			sendMsg("@"+person+" Balance : "+balance+ "Doge ("+pending_balance+" Doge)",chatid)
```

Comment:
```suggestion
			sendMsg("@"+person+" Balance : "+math.floor(balance)+ "Doge ("+math.floor(pending_balance)+" Doge)",chatid)
```

Another example of something I fixed in `master` thats getting undone and overwritten.